src/commands/hello/org.ts

```diff
@@ -14,7 +14,7 @@ Messages.importMessagesDirectory(__dirname);
 
 // Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
 // or any library that is using the messages framework can also be loaded this way.
-const messages = Messages.loadMessages('<REPLACE_ME>', 'org');
+const messages = Messages.loadMessages('@salesforce/plugin-template', 'org');
```

Comment:
I think this should stay `<REPLACE_ME>`

Reply:
if we keep it as <REPLACE_ME> the test won't pass and it won't compile. It's hard to use the dev-config and use this as a template with real code in it. We can just do `--no-verify` when committing and pushing to this repo too